File: FirstApp/ServerManagement/webjs.vb
Imports System.IO
Imports System.Net

Public Class webjs
    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        UpdateImage()
    End Sub

    Public Sub UpdateImage()
        Dim req As WebRequest = WebRequest.Create($"http://{My.Settings.connection_url}?purpose=recent_image")
        req.Method = "POST"
        Using request As WebResponse = req.GetResponse
            Using stream As Stream = request.GetResponseStream
                ImageBox.Image = New Bitmap(Image.FromStream(stream))
            End Using
        End Using
    End Sub

End Class

File: FirstApp/ServerManagement/webjs.cs
using System;
using System.Drawing;
using System.Net;

namespace FirstApp
{

    public partial class webjs
    {
        public webjs()
        {
            InitializeComponent();
        }
        private void Button1_Click(object sender, EventArgs e)
        {
            UpdateImage();
        }

        public void UpdateImage()
        {
            var req = WebRequest.Create($"http://{My.MySettingsProperty.Settings.connection_url}?purpose=recent_image");
            req.Method = "POST";
            using (var request = req.GetResponse())
            {
                using (var stream = request.GetResponseStream())
                {
                    ImageBox.Image = new Bitmap(Image.FromStream(stream));
                }
            }
        }

    }
}